content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+
+***Checklist***
+- [ ] My output address is _________________. I want my staked funds(when unstaked) and rewards here. I have backed it up safely and I will not share the privatekeys/keyfile of this address ever.
+
+##  Information to get from your Node Operator:
+
+***Checklist***
+- [ ] Operator Address:  __________________________________
+- [ ] Operator Public Key:  __________________________________ 
+- [ ] Chains your Nodes Operator will support:  __________________________________
+- [ ] Domain:  __________________________________
+
+## Preparing the Output Address
+
+1. Login to your  output address using the pocket official wallet below. Note down the password used to create the wallet. https://wallet.pokt.network/
+2. **IMPORTANT:** Backup the keyfile.json using “export keyfile” and the password. Lose this and you will lose your funds. Then send 15011 tokens to this address. 15010 will be staked, leaving one for transaction fees.(Each transaction costs 0.01 pokt at the time this doc is written)
+ 
+![](/images/wallet.png)
+
+## Staking Non-Custodial
+
+1. Login from output address
+2. Go to the staking Tab: https://wallet.pokt.network/staking
+3. You should already have the operator public key, output address,chains and domain from the checklists. 
+
+{{% notice style="warning" %}}
+**1. Wrong output address will result in permanent loss of funds**
+
+**2. Wrong operator public key may lock your funds for 21 days(you have to unstake to get your funds back)**
+{{% /notice %}}
+
+![](/images/wallet-staking.png)
+
+4. Hit the Stake Node Button, verify your stake on poktscan in 15-20 minutes.
+
+***Checklist***
+- [ ] I verified that it is staked non-custodial on poktscan at the url https://poktscan.com/node/\<myoperatoraddress\> with the correct output address.
+   
+
+## Managing Non-Custodial Nodes (Upstake)
+1. Once you are confident that you did it right, you may upstake your node for example from 15010 to 60010.
+2. Login using the output address, your output address must have 45001 tokens
+3. Fill out the same [form](/node/trustlessstaking/#staking-non-custodial) to stake but enter 60010 not 15010.(60010-15010 = 45000 is the amount to “upstake”)
+
+## Managing Non-Custodial Nodes (Unstake)
+1. Login from Output address
+2. Go to manage non custodial Nodes
+3. Enter Operator Address
+4. Hit Unstake 
+
+![](/images/wallet-unstaking.png)
+
+## FAQs
+**If the node operator unstakes from the operator keys where will the staked funds go?**
+No matter who unstakes all staked funds will come to the output address in 21 days.
+
+**Can the output address be changed once staked?**
+Neither the operator address nor the output address can do any transaction to edit the output address. Therefore be very careful to back up your output address keys and set it correctly on a node.
```

Comment:
The output address cannot be changed once staked; ensure it's correct and backed up.